### ISSUE TITLE
Local ember version not updated after using global ember

### ISSUE THREAD

**Issue by diofinus** [artificialio/sane#181]
After update the sane to version `sane-cli@0.1.0-beta.2` I try to use global ember which is `1.13.1` but when I do sane up the ember version that shown is `0.2.7` which is my local ember at sane.

**Comment by IanVS** [comment]
This is because sane now defaults to using the local version of ember rather than global.  It asks the first time when you're setting up the project, but then uses the local copy from then out.  

For now, you should be able to either:

- Delete the local ember out of the project package.json and node_modules (untested)
- Upgrade the local version of ember to match your global


These are just stop-gaps until a better solution can be implemented.

**Comment by IanVS** [comment]
Actually after thinking about this some more, I'm not convinced that the current behavior is incorrect.  If you want to upgrade the version of ember-cli in your app, you should do so by following the recommended upgrade process.  Or am I missing something?

**Comment by diofinus** [comment]
Yes, right now I also do the recommended upgrade process to upgrade my ember-cli in my app
